_spinnaker/terraform_integration.md

```diff
@@ -15,131 +15,26 @@ If you decide to enable this feauture and have any feedback you'd like to submit
-
-The core of our Terraform integration is the Terraformer service. This is the service which is responsible for fetching your Terraform code and executing it. To do this, we'll need to install the following Kubernetes manifest using `kubectl`. *Note: we'll be adding support for deploying Terraformer via Halyard in future releases.* 
-
-You'll need to replace 4 references before doing so:
-1. `your-github-api-token` - the API token which will be used to fetch your repository
-2. `profile-name` - the name of the AWS profile which your Terraform code will use to provision infrastructure
-3. `your-aws-access-key` - the AWS Access Key ID of your keypair
-4. `your-aws-secret-access=key` - the AWS Secret Access Key of your keypair
+### Enabling the Terraform integration
 
+You can enable the Terraform integration via Armory Halyard. Before you do, be sure you have a Github access token. This will enable the Terraform integration to interact with entire Github repositories.
 ```
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: terraformer-config
-  namespace: spinnaker
-data:
-  terraformer.yml: |
-    server:
-      host: 0.0.0.0
-      port: 7088
-    redis:
-      enabled: ${services.redis.enabled}
-      host: ${services.redis.address}
-      port: ${services.redis.port}
-    clouddriver:
-      baseUrl: http://spin-clouddriver:7002
-    git:
-      token: {your-github-api-token}
-    executor:
-      workers: 3
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: terraformer-credentials
-  namespace: spinnaker
-data:
-  credentials: |
-    [profile-name]
-    aws_access_key_id = {your-aws-access-key}
-    aws_secret_access_key = {your-aws-secret-access-key}
----
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: spin-terraformer
-  namespace: spinnaker
-  labels:
-    app: spin
-    cluster: spin-terraformer
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: spin
-      cluster: spin-terraformer
-  template:
-    metadata:
-      labels:
-        app: spin
-        cluster: spin-terraformer
-    spec:
-      containers:
-      - name: terraformer
-        image: docker.io/armory/terraformer:0.0.1-master-cc65a45
-        ports:
-        - containerPort: 7088
-        volumeMounts:
-        - name: terraformer-config
-          mountPath: /opt/spinnaker/config
-        - name: terraformer-credentials
-          mountPath: /root/.aws
-      volumes:
-      - name: terraformer-config
-        configMap:
-          name: terraformer-config
-      - name: terraformer-credentials
-        configMap:
-          name: terraformer-credentials
----
-kind: Service
-apiVersion: v1
-metadata:
-  name: spin-terraformer
-  namespace: spinnaker
-  labels:
-    app: spin
-    cluster: spin-terraformer
-spec:
-  selector:
-    app: spin
-    cluster: spin-terraformer
-  ports:
-  - protocol: TCP
-    port: 7088
-    targetPort: 7088
-  type: ClusterIP
+$ hal armory terraform edit --git-enabled true --git-token
```

Comment:
Is this command right? I was having a problem with it accepting the `true` after `git-enabled`. It seemed to just want the `git-enabled` flag. Also no value for `git-token` in this example.

Reply:
Nice catch! Will fix.